packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1032,6 +1034,11 @@ export class ContainerRuntime
 	 */
 	private nextSummaryNumber: number;
 
+	/**
+	 * If true, loading or using a Tombstoned object should merely log, not fail
+	 */
+	public readonly disableGcTombstoneEnforcement: boolean;
```

Comment:
Can we move this to garbage collector and expose it on IGarabgeCollector. That way this logic remains encapsulated in garbage collector.
This can then become a getter than gets the value from garbage collector.